src/components/WeeklyRoutinePage/WeekDayCard.css

```diff
@@ -127,14 +127,14 @@
   justify-content: center;
   align-items: center;
   margin: 20px 0;
+  z-index: 5;
```

Comment:
z-index only works with absolutely positioned elements. Is it doing something?

Reply:
without it, my button shows up like this

![Screenshot 2021-05-04 at 21 43 56](https://user-images.githubusercontent.com/78050731/117060918-d55b7980-ad21-11eb-8559-ac8b35d32120.png)